### DISCUSSION TITLE
how to let super table scrollable

### DISCUSSION THREAD

**Discussion by ahmedelnaqa** [Budibase/budibase#13690] [Help] [answered]
Dears, 

I am using Super Table but I didn't find a  way to let it a scrollable all the columns in the mobile shown in the screen and move hidden data on it.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @ahmedelnaqa,

For this specific question you would be better off reaching out to the developer of the [super table](https://github.com/poirazis/bb-component-SuperTable) plugin. Budibase didn't create that plugin nor do they maintain the plugin so your question would be better as an issue in the plugin's repository.
